examples/meta/generator/targets/python.json

```diff
@@ -71,13 +71,15 @@
         "MethodCall": {
             "Default": "$object.$method($arguments)",
             "get_int": "$object.get($arguments)",
+            "get_long": "$object.get($arguments)",
             "get_int_vector": "$object.get($arguments)",
             "get_real": "$object.get($arguments)",
             "get_real_vector": "$object.get($arguments)",
             "get_real_matrix": "$object.get($arguments)",
             "get_char_string_list": "$object.get($arguments)",
             "get_word_string_list": "$object.get($arguments)",
-            "get_option": "$object.get($arguments)"
+            "get_option": "$object.get($arguments)",
+            "get_string": "$object.get($arguments)"
```

Comment:
you will need to update the other .json files as well (see CI)

Reply:
The CI's errors are unrelated to this (the python issue will be fixed here #4626). Actually, I have no idea why the build process fails on MacOS :/

Reply:
ah actually I was talking nonsense, since `get_string` is part of swig so all other langs will use exactly that call, nervermind :)